tests/test_download_shieldhit.py

```diff
@@ -4,31 +4,31 @@
 import sys
 import pytest
 from pathlib import Path
-from yaptide.admin.simulators import download_shieldhit_from_s3
+from yaptide.admin.simulator_storage import download_shieldhit_from_s3
 
 
 def check_if_environment_variables_set() -> bool:
     """Check if environment variables are set"""
     result = True
-    if not 'S3_ENDPOINT' in os.environ:
-        result = False
-    if not 'S3_ACCESS_KEY' in os.environ:
-        result = False
-    if not 'S3_SECRET_KEY' in os.environ:
-        result = False
-    if not 'S3_SHIELDHIT_BUCKET' in os.environ:
-        result = False
-    if not 'S3_SHIELDHIT_KEY' in os.environ:
-        result = False
+    for var_name in ['S3_ENDPOINT', 'S3_ACCESS_KEY', 'S3_SECRET_KEY', 'S3_SHIELDHIT_BUCKET', 'S3_SHIELDHIT_KEY']:
+        if var_name not in os.environ:
+            logging.error(' variable %s not set', var_name)
```

Comment:
Removing probably not wanted space at the beginning
```suggestion
            logging.error('variable %s not set', var_name)
```